Software 0.29.4 release notes (#1190)
astronomer/docs: software_versioned_docs/version-0.29/release-notes.md

0.30 is the latest long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To review release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).

Astronomer tests all Astronomer Software versions for scale, reliability, and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).

## v0.29.4

Release date: September 13, 2022

### Additional improvements

> - You can now specify `authUrlParams` for your identity provider (IdP) in `config.yaml`.

Author: ```suggestion
- You can now specify `authUrlParams` for your identity provider (IdP) in `config.yaml`
```

> We're committed to testing all Astronomer Software versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).

Reviewer: ```suggestion
Astronomer tests all Astronomer Software versions for scale, reliability, and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
```
